project/src/components/sorting-form/sorting-form.tsx

```diff
@@ -1,18 +1,74 @@
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { SortType } from '../../const';
+import { changeSortType } from '../../store/action';
+import { useState, useRef } from 'react';
+import { useOnClickOutside } from 'usehooks-ts';
+import cn from 'classnames';
+
 function SortingForm(): JSX.Element {
+  const [isOptionsOpen, setOptionsView] = useState<boolean>(false);
```

Comment:
Нейминг можно короче, если блок такой в компоненте один то я обычно ограничиваюсь 
```suggestion
  const [open, setOpen] = useState<boolean>(false);
```